library/proc_macro/src/bridge/client.rs

```diff
@@ -277,6 +338,12 @@ struct Bridge<'a> {
     /// Server-side function that the client uses to make requests.
     dispatch: closure::Closure<'a, Buffer, Buffer>,
 
+    /// Server-side function to validate and normalize an ident.
+    validate_ident: extern "C" fn(buffer::Slice<'_>, &mut Buffer) -> bool,
+
+    /// Interned store for storing symbols within the client.
+    symbols: handle::InternedStore<Rc<str>>,
```

Comment:
Random thought: since this uses `HashMap` without the `Fx` hash the compiler uses, it might be significantly slower to hash? Maybe switching hashers can help (though if it can help with the existing handle stores, we might want to land that as a separate PR).